frontend/src/components/associations/library/home/AssociationLibraryHome.tsx

```diff
@@ -8,38 +8,123 @@ import Row from "react-bootstrap/Row";
 import { LoanableCard } from "./LoanableCard";
 import Col from "react-bootstrap/Col";
 import { PageTitle } from "../../../utils/PageTitle";
+import { SidebarSeparator, SidebarSpace } from "../../../utils/sidebar/Sidebar";
+import { SidebarInputSearch } from "../../../utils/sidebar/SidebarInputSearch";
+import { Instructions } from "../../../utils/Instructions";
+import { Link } from "react-router-dom";
+import { CheckboxField } from "../../../utils/sidebar/CheckboxField";
+import { SidebarSection } from "../../../utils/sidebar/SidebarSection";
 
 export const AssociationLibraryHome = ({
   association,
 }: {
   association: Association;
 }) => {
+  const [searchParams, setSearchParams] = useState({});
+  const [statusParams, setStatusParams] = useState({
+    status: ["AVAILABLE", "REQUESTED"],
+  });
+
   return (
-    <AssociationLayout association={association}>
+    <AssociationLayout
+      association={association}
+      additionalSidebar={
+        <>
+          <SidebarSeparator />
+          <SidebarInputSearch
+            setParams={setSearchParams}
+            placeholder="Chercher un objet"
+          />
+          <SidebarSpace />
+          <SidebarSection
+            title="Voir les objets..."
+            retractable={false}
+            retractedByDefault={false}
+          >
+            {/*
+              In this component, the AVAILABLE and REQUESTED status are treated
+              as the same (“Disponible”).
+               */}
+            <CheckboxField
+              label={"Disponibles"}
+              state={
+                statusParams.status.includes("AVAILABLE") &&
+                statusParams.status.includes("REQUESTED")
+              }
+              onChange={(ticked) =>
+                setStatusParams((oldStatus) => ({
+                  status: ticked
+                    ? [...oldStatus.status, "AVAILABLE", "REQUESTED"]
+                    : oldStatus.status.filter(
+                        (x) => x !== "AVAILABLE" && x !== "REQUESTED"
```

Comment:
If I remember well, on there is somewhere a `priority` mixin because we can potentially have multiple requests for the same Loanable at the same time. But here a Loanable is considered unavailable as soon it's requested, so the backend and the frontend are not on the same page here

Reply:
It's the opposite here: keeping the “Disponible” loanables will send a request for filtering on the AVAILABLE and REQUESTED loanables